inblay/report/purchase_order_templates.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <template id="report_purchaseorder_document_inherit" inherit_id="purchase.report_purchaseorder_document">
+        <xpath expr="//div[@id='informations']/div[last()]" position="after">
```

Comment:
Could this be used instead?

`expr="//div[@id='informations']/div[@t-if='o.date_order']"`

Reply:
Done